fusor-ember-cli/README.md

```diff
@@ -1,5 +1,71 @@
 # fuser-ember-cli
 
-This directory is a copy of the files in the repo [fusor-ember-cli](https://github.com/fusor/fusor-ember-cli/).
+This repo is the development environment to create [FusorUi](https://github.com/fusor/fusor/ui/) which a rails engine that is added to [Foreman](https://github.com/theforeman/foreman/).
+
+[FusorUi](https://github.com/fusor/ui/) is essentially the output from this repo which is an [ember-cli](http://www.ember-cli.com/) project that contains assets such as javascript, stylesheets, images, and fonts.
+
+## Important Note
+
+The fuser-ember-cli/dist directory is the [.gitignore](https://github.com/fusor/fusor/blob/master/.gitignore) so you wont find it in this repository.
+
+The fuser-ember-cli/dist distory is generated automatically by [ember-cli](http://www.ember-cli.com/) when you run `ember server` or `ember build` locally inside the this directory.
+
+## Workflow
+
+1. Clone [fusor](https://github.com/fusor/fusor/) to your local workstation.
+2. `cd fusor-ember-cli`
+3. Run `ember server`
+4. Prepare for development
+  - In [controllers/application.js](https://github.com/fusor/fusor/blob/master/fusor-ember-cli/app/controllers/application.js#L8), change `deployAsPlugin` from `true` to `false`
+  - In [styles/app.scss](https://github.com/fusor/fusor/tree/master/fusor-ember-cli/app/styles/app.scss#L3), change `padding-top` from `40px` to `0px`
```

Comment:
These two steps seems unnecessary, cold you script that away? Leaving this to a human is very error prone.